stat.proto

```diff
@@ -120,4 +120,7 @@ message StatJourneySection {
     optional string     to_admin_insee = 29;
 }
 
-
+message Pagination {
+    repeated StatRequest requests = 1;
```

Comment:
My bad, I change Pagination to StatResponse. And I add relation between it in StatRequest (root of tree).